colossalai/moe/manager.py

```diff
@@ -140,6 +140,10 @@ def get_info(self, num_experts: int, use_tp: bool = False) -> Tuple[int, MoePara
 
         if not (ep_size in self.parallel_info_dict):
             self.parallel_info_dict[ep_size] = get_moe_info(ep_size, dp_size, pp_size, ep_inside=self.use_ep_inside)
+            if self.use_ep_inside:
+                print(f"MoE Parallel: pp {pp_size}, dp {dp_size}, ep {ep_size}")
```

Comment:
Please ensure this is only printed on one process

Reply:
done